client/web/compose/src/components/Public/Record/CustomFilterPreset/index.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div>
+    <b-modal
+      :visible="showModal"
+      :title="$t('recordList.presetFilter.addFilterPresetName')"
+      body-class="p-0"
+      footer-class="flex-column align-items-stretch"
+      centered
+      @hide="onModalHide"
+    >
+      <b-card
+        class="pt-0"
+      >
+        <b-form-group
+          :label="$t('recordList.presetFilter.filterName')"
+        >
+          <b-form-input
+            v-model="filterName"
+            required
```

Comment:
the required here doesn't have any effect, you should however prevent saving(button should be disabled) if there is no filter name. You can also indicate its required by having a * in the label